cmd/secret/main.go

```diff
@@ -1,14 +1,16 @@
 package main
 
 import (
+	"context"
 	"fmt"
-)
 
-var (
-	revision = "unknown"
```

Comment:
Do not remove this one: this is for version. Pass it to `cobra.Command.Version` field.